docs/docusaurus/docs/core/connect_to_data/dataframes/dataframes.md

```diff
@@ -75,124 +76,210 @@ Because the dataframes reside in memory you do not need to specify the location
+- Optional. A Validation Definition.
+
+### Procedure
+
+1. Define the Batch Parameter dictionary.
+
+   A dataframe can be added to a Batch Parameter dictionary by defining it as the value of the dictionary key `dataframe`:
+
+   ```python title="Python" name="docs/docusaurus/docs/core/connect_to_data/dataframes/_examples/_batch_parameters_batch_definition.py - batch parameters example"
+   ```
+
+   The following examples create a dataframe by reading a `.csv` file and stores it in a Batch Parameter dictionary:
+
    <Tabs queryString="execution_engine" groupId="execution_engine" defaultValue='pandas'>
 
       <TabItem value="pandas" label="pandas">
 
-      ```Python title="Python" name="docs/docusaurus/docs/core/connect_to_data/dataframes/_examples/_pandas_df_data_asset.py - full example"
+      ```Python title="Python" name="docs/docusaurus/docs/core/connect_to_data/dataframes/_examples/_batch_parameters_batch_definition.py - pandas dataframe"
       ```
 
       </TabItem>
 
       <TabItem value="spark" label="Spark">
 
-      ```Python title="Python" name="docs/docusaurus/docs/core/connect_to_data/dataframes/_examples/_spark_df_data_asset.py - full example"
+      ```Python title="Python" name="docs/docusaurus/docs/core/connect_to_data/dataframes/_examples/_batch_parameters_batch_definition.py - spark dataframe"
       ```
 
       </TabItem>
 
    </Tabs>
 
-</TabItem>
+3. Pass the Batch Parameter dictionary to a `get_batch()` or `validate()` method call.
 
-</Tabs>
+   Runtime Batch Parameters can be provided to the `get_batch()` method of a Batch Definition or to the `validate()` method of a Validation Definition.
+
+   <Tabs>
+
+   <TabItem value="batch" label="Batch Definition">
+
+   The `get_batch()` method of a Batch Definition retrieves a single Batch of data.  Runtime Batch Parameters can be provided to the `get_batch()` method to specify the data returned as a Batch.  The `validate()` method of this Batch can then be used to test individual Expectations. 
+
+   ```Python title="Python" name="docs/docusaurus/docs/core/connect_to_data/dataframes/_examples/_batch_parameters_batch_definition.py - batch.validate() example"
+   ```
+
+   The results generated by `batch.validate()` are not persisted in storage.  This workflow is solely intended for interactively creating Expectations and engaging in data Exploration.
+
+   For further information on using an individual Batch to test Expectations see [Test an Expectation](/core/define_expectations/test_an_expectation.md).
+
+   </TabItem>
+
+   <TabItem value="validate" label="Validation Definition">
+
+   A Validation Definition's `validate()` method runs an Expectation Suite against a Batch returned by a Batch Definition.  Runtime Batch Parameters can be provided to a Validation Definition's `validate()` method to specify the data returned in the Batch.  This allows you to validate your dataframe by executing the Expectation Suite included in the Validation Definition.
```

Comment:
`validate()` should be `run()` in this paragraph.

Reply:
Fixed!  Thank you.